code/game/machinery/vending.dm

```diff
@@ -1464,7 +1477,7 @@ var/global/num_vending_terminals = 1
 	pack = /obj/structure/vendomatpack/security
 
 /obj/machinery/vending/security/used
-	req_access_txt = 0
+	req_access = "0"
```

Comment:
If you want that changed let  me know what you want.  It doesn't runtime and it works as intended.